lib/src/migration_visitor.dart

```diff
@@ -67,9 +73,14 @@ abstract class MigrationVisitor extends RecursiveAstVisitor {
 
   /// Visits the stylesheet at [dependency], resolved relative to [source].
   @protected
-  void visitDependency(Uri dependency, Uri source) {
-    var stylesheet = parseStylesheet(source.resolveUri(dependency));
-    visitStylesheet(stylesheet);
+  void visitDependency(Uri dependency, Uri source, [FileSpan context]) {
+    var url = source.resolveUri(dependency);
+    var stylesheet = parseStylesheet(url);
+    if (stylesheet != null) {
+      visitStylesheet(stylesheet);
+    } else {
+      _missingDependencies[url] = context;
```

Comment:
```suggestion
      _missingDependencies.putIfAbsent(url, () => context);
```

This is minor, but I think it's more likely that an earlier import is more relevant to the user.